boards/weact/mini_stm32h743/mini_stm32h743_defconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Charles Dias <charlesdias.cd@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+# Enable console
+CONFIG_CONSOLE=y
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+# Enable clocks
+CONFIG_CLOCK_CONTROL=y
+
+# Enable pin controller
+CONFIG_PINCTRL=y
+
+# Enable SPI
+CONFIG_SPI=y
+
+# Enable SPI driver for ST7735R
+CONFIG_ST7735R=y
```

Comment:
This can't be part of default board configuration

Reply:
Fixed. I removed and create the `samples/subsys/display/lvgl/boards/mini_stm32h743.conf` file to work with the LVGL sample.

Thanks for your review!